src/lib/parts.ts

```diff
@@ -66,8 +78,7 @@ export class AttributeCommitter {
       const part = this.parts[i];
       if (part !== undefined) {
         const v = part.value;
-        if (v != null &&
-            (Array.isArray(v) || typeof v !== 'string' && v[Symbol.iterator])) {
+        if (isIterable(v)) {
```

Comment:
this is a potentially hot code path... is this just for readability?

Reply:
Fair, reverted.